v1.1.0-dev1/conformance_test_v1.1.0-dev1.yaml

```diff
@@ -87,7 +87,7 @@
       size: 84
       location: error.txt
 
-- job:
+- job: v1.1.0-dev1/empty.json
   tool: v1.1.0-dev1/egrep-stderr-shortcut.cwl
   doc: Test command line with stderr redirection, brief syntax
```

Comment:
I don't know why this test is failing.  However, these tests are a bit unstable in that they rely on egrep printing out a specific usage message on stderr.  I am updating these tests to use "echo 2>&1" to print a message to stderr.